patches/build-util-android_chrome_version.py.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
-index b50847f4de283a0a409bd5abe5f32800e9083084..f099941b29bed1297888320dffd694a7338c056a 100755
+index b50847f4de283a0a409bd5abe5f32800e9083084..97a74bb9dab3c09f944bc9875877b9f657d6fdce 100755
 --- a/build/util/android_chrome_version.py
 +++ b/build/util/android_chrome_version.py
-@@ -314,7 +314,7 @@ def GenerateVersionCodes(version_values, arch, is_next_build):
+@@ -186,6 +186,7 @@ def _GetAbisToDigitMask(build_number):
+     mapped to version code suffix.
+   """
+ 
++  if build_number >= 53 : build_number = 5750 # we want to introduce this change starting from 1.53.x
```

Comment:
why are we doing something different from upstream here? The issue does not provide any useful information for this change.

Reply:
if we need https://chromium.googlesource.com/chromium/src/+/03c3f83bc5f51df8c549f3852d159c084ba0a8a0%5E%21/#F0 the issue should specify the reason we need it and we should pull in the upstream patch as-is

Reply:
We are following upstream change here, the problem is that we override build number and thus new version codes won't be applied, since our build number is unlikely to ever get to 5750